services/opentsdb/service.go

```diff
@@ -216,7 +216,7 @@ func (s *Service) handleTelnetConn(conn net.Conn) {
 		tags := make(map[string]string)
 		for t := range tagStrs {
 			parts := strings.SplitN(tagStrs[t], "=", 2)
-			if len(parts) != 2 {
+			if len(parts) != 2 || len(parts[0]) == 0 || len(parts[1]) == 0 {
```

Comment:
Super crazy smal nit: For code clarity, I prefer `parts[0] == ""`.  The compiler is at some point going to compile `len(parts[0]) == 0` the same way it compiles `parts[0] == ""` so I prefer code clarity for this.